projects/bazel-rules-fuzzing-test/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+declare -r QUERY='
```

Comment:
Since this is test project, it is good to add comments on each step in this file. Can you do this in next PR. Also, need to remember to add docs on subsection of https://google.github.io/oss-fuzz/getting-started/new-project-guide/

Reply:
> Since this is test project, it is good to add comments on each step in this file. Can you do this in next PR. 

I've just seen this. Let me do this in #5039.

> Also, need to remember to add docs on subsection of https://google.github.io/oss-fuzz/getting-started/new-project-guide/

Good point, let me do this in a subsequent PR. I'm planning to overhaul documentation in the `rules_fuzzing` library, too.